Traceback formatter test failure

```
  1) Notebook trace formatter
       ipython: 8.3.0, ipykernel: 6.13.0:

      AssertionError: expected [ …(4) ] to deeply equal [ …(4) ]
      + expected - actual

       [
         "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m"
         "\u001b[0;31mNameError\u001b[0m                                 Traceback (most recent call last)"
      -  "\u001b[1;32mUsers\\USR\\notebooks\\error.ipynb Cell 1\u001b[0m in \u001b[0;36m<cell line: 2>\u001b[0;34m()\u001b[0m\n\u001b[1;32m      <a href='vscode-notebook-cell:/Users/USR/notebooks/error.ipynb#W0sZmlsZQ%3D%3D?line=0'>1</a>\u001b[0m \u001b[39m1\u001b[39m\n\u001b[0;32m----> <a href='vscode-notebook-cell:/Users/USR/notebooks/error.ipynb#W0sZmlsZQ%3D%3D?line=1'>2</a>\u001b[0m x\n"
      +  "\u001b[1;32m/Users/USR/notebooks/error.ipynb Cell 1\u001b[0m in \u001b[0;36m<cell line: 2>\u001b[0;34m()\u001b[0m\n\u001b[1;32m      <a href='vscode-notebook-cell:/Users/USR/notebooks/error.ipynb#W0sZmlsZQ%3D%3D?line=0'>1</a>\u001b[0m \u001b[39m1\u001b[39m\n\u001b[0;32m----> <a href='vscode-notebook-cell:/Users/USR/notebooks/error.ipynb#W0sZmlsZQ%3D%3D?line=1'>2</a>\u001b[0m x\n"
         "\u001b[0;31mNameError\u001b[0m: name 'x' is not defined"
       ]
      
      at Context.<anonymous> (out\notebooks\outputs\tracebackFormatter.unit.test.js:511:19)
      at processImmediate (node:internal/timers:466:21)

  2) Notebook trace formatter
       ipython 8.5.0, ipykernel 6.16.0:

      AssertionError: expected [ …(4) ] to deeply equal [ …(4) ]
      + expected - actual

       [
         "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m"
         "\u001b[0;31mNameError\u001b[0m                                 Traceback (most recent call last)"
      -  "\u001b[1;32mUsers\\USR\\notebooks\\error.ipynb Cell 1\u001b[0m in \u001b[0;36m2\n\u001b[1;32m      <a href='vscode-notebook-cell:/Users/USR/notebooks/error.ipynb#W0sZmlsZQ%3D%3D?line=0'>1</a>\u001b[0m \u001b[39m1\u001b[39m\n\u001b[0;32m----> <a href='vscode-notebook-cell:/Users/USR/notebooks/error.ipynb#W0sZmlsZQ%3D%3D?line=1'>2</a>\u001b[0m x\n"
      +  "\u001b[1;32m/Users/USR/notebooks/error.ipynb Cell 1\u001b[0m in \u001b[0;36m2\n\u001b[1;32m      <a href='vscode-notebook-cell:/Users/USR/notebooks/error.ipynb#W0sZmlsZQ%3D%3D?line=0'>1</a>\u001b[0m \u001b[39m1\u001b[39m\n\u001b[0;32m----> <a href='vscode-notebook-cell:/Users/USR/notebooks/error.ipynb#W0sZmlsZQ%3D%3D?line=1'>2</a>\u001b[0m x\n"
         "\u001b[0;31mNameError\u001b[0m: name 'x' is not defined"
       ]
      
      at Context.<anonymous> (out\notebooks\outputs\tracebackFormatter.unit.test.js:521:19)
      at processImmediate (node:internal/timers:466:21)
`~

--- a/src/notebooks/outputs/tracebackFormatter.unit.test.ts
+++ b/src/notebooks/outputs/tracebackFormatter.unit.test.ts
@@ -19,7 +19,7 @@ suite(`Notebook trace formatter`, function () {
         when(cell.index).thenReturn(0);
         when(cell.notebook).thenReturn(instance(notebook));
         document = mock<TextDocument>();
-        const uri = Uri.parse('vscode-notebook-cell:/Users/USR/notebooks/error.ipynb#W0sZmlsZQ==');
+        const uri = Uri.parse('vscode-notebook-cell:error.ipynb#W0sZmlsZQ==');
         when(document.uri).thenReturn(uri);
         when(cell.document).thenReturn(instance(document));
     });
@@ -46,7 +46,7 @@ suite(`Notebook trace formatter`, function () {
         const expected = [
             '[0;31m---------------------------------------------------------------------------[0m',
             '[0;31mNameError[0m                                 Traceback (most recent call last)',
-            "[1;32m/Users/USR/notebooks/error.ipynb Cell 1[0m in [0;36m<cell line: 2>[0;34m()[0m\n[1;32m      <a href='vscode-notebook-cell:/Users/USR/notebooks/error.ipynb#W0sZmlsZQ%3D%3D?line=0'>1</a>[0m [39m1[39m\n[0;32m----> <a href='vscode-notebook-cell:/Users/USR/notebooks/error.ipynb#W0sZmlsZQ%3D%3D?line=1'>2</a>[0m x\n",
+            "[1;32merror.ipynb Cell 1[0m in [0;36m<cell line: 2>[0;34m()[0m\n[1;32m      <a href='vscode-notebook-cell:error.ipynb#W0sZmlsZQ%3D%3D?line=0'>1</a>[0m [39m1[39m\n[0;32m----> <a href='vscode-notebook-cell:error.ipynb#W0sZmlsZQ%3D%3D?line=1'>2</a>[0m x\n",
             "[0;31mNameError[0m: name 'x' is not defined"
         ];
 
@@ -66,7 +66,7 @@ suite(`Notebook trace formatter`, function () {
         const expected = [
             '[0;31m---------------------------------------------------------------------------[0m',
             '[0;31mNameError[0m                                 Traceback (most recent call last)',
-            "[1;32m/Users/USR/notebooks/error.ipynb Cell 1[0m in [0;36m2\n[1;32m      <a href='vscode-notebook-cell:/Users/USR/notebooks/error.ipynb#W0sZmlsZQ%3D%3D?line=0'>1</a>[0m [39m1[39m\n[0;32m----> <a href='vscode-notebook-cell:/Users/USR/notebooks/error.ipynb#W0sZmlsZQ%3D%3D?line=1'>2</a>[0m x\n",
+            "[1;32merror.ipynb Cell 1[0m in [0;36m2\n[1;32m      <a href='vscode-notebook-cell:error.ipynb#W0sZmlsZQ%3D%3D?line=0'>1</a>[0m [39m1[39m\n[0;32m----> <a href='vscode-notebook-cell:error.ipynb#W0sZmlsZQ%3D%3D?line=1'>2</a>[0m x\n",
             "[0;31mNameError[0m: name 'x' is not defined"
         ];
 

